ui/src/pages/backoffice/Acquisition/Vendor/VendorEditor/components/VendorForm/VendorForm.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import pick from 'lodash/pick';
+import { StringField, TextField } from '@forms';
+import { vendor as vendorApi } from '@api/acquisition/vendor';
+import { AcquisitionRoutes } from '@routes/urls';
+import { goTo } from '@history';
+import { BaseForm } from '@forms';
+
+export class VendorForm extends Component {
+  prepareData = data => {
+    return pick(data, ['name', 'address', 'email', 'phone', 'notes']);
+  };
+
+  create = data => {
```

Comment:
Rename potentially to `createVendor`, just to keep the name convention as in the other forms?

Reply:
I noticed in some places is explicit and some others not, and my thinking was I am inside the vendor form so obviously will create and update vendor, by having a look around I see that pretty in all the `*Form.js` files these methods are repeated and are the same. Maybe it would make sense to move the to BaseForm.js and provide the relevant api module as a prop.

Reply:
If there inconsistencies we could normalise them as well, but honestly I wouldn't move them from here as they are just simple functions and I am not sure it's worth the extra boilerplate for this...Even if "software engineering" speaking is tempting :)

Reply:
I agree, it's more straightforward/clear to see where it comes from rather than start looking for who supplied the prop and what prop.